nonebot_plugin_chatgpt/chatgpt.py

```diff
@@ -38,20 +36,54 @@ def __init__(
         self.api_url = api
         self.proxies = proxies
         self.timeout = timeout
-        self.authorization = ""
-        self.conversation_id = None
+        self.content = None
+        self.authorization = None
         self.parent_id = None
-
-        self.cf_clearance = ""
-        self.user_agent = ""
-
+        self.conversation_id = None
+        self.browser = None
+        self.playwright = async_playwright()
         if self.session_token:
             self.auto_auth = False
         elif self.account and self.password:
             self.auto_auth = True
         else:
             raise ValueError("至少需要配置 session_token 或者 account 和 password")
 
+    async def playwright_start(self):
+        # 启动浏览器，在插件开始运行时调用
```

Comment:
```suggestion
        """启动浏览器，在插件开始运行时调用"""
```
如果要作为函数的注释，应该使用文档字符串，其他的几个注释同样